h2o-algos/src/main/java/hex/schemas/Word2VecSynonymsV3.java

```diff
@@ -0,0 +1,19 @@
+package hex.schemas;
+
+import water.Iced;
+import water.api.API;
+import water.api.schemas3.KeyV3;
+import water.api.schemas3.SchemaV3;
+
+public class Word2VecSynonymsV3 extends SchemaV3<Iced, Word2VecSynonymsV3> {
+  @API(help="Source word2vec Model", required = true, direction = API.Direction.INPUT)
+  public KeyV3.ModelKeyV3 model;
+  @API(help="Target word", required = true, direction = API.Direction.INPUT)
+  public String target;
```

Comment:
Will change to "word", "target" was a leftover from the previous implementation.

We could also accept an array of words, it would be a natural extension of this function. For now I wanted to keep things simple and on par with Spark's word2vec.